app/models/feed.rb

```diff
@@ -19,7 +19,7 @@ class Feed < ApplicationRecord
   serialize :exclude_tags, JSON
   serialize :audio_format, HashSerializer
 
-  belongs_to :podcast, -> { with_deleted }, optional: true
+  belongs_to :podcast, -> { with_deleted }, optional: true, touch: true
```

Comment:
At some point, we'll need to go through and make sure saves are cascading up to the podcast/episode `updated_at` correctly.  If we want any hope of adding caching.

Reply:
is there anything that currently calls `feed.touch`? or is a `.save` supposed to trigger this as well?